test/gst-vaapi/encode/vp9.py

```diff
@@ -82,13 +82,13 @@ def init(self, tspec, case, gop, bitrate, fps, refmode, looplvl, loopshp):
       refmode   = refmode,
     )
  
-  @slash.parametrize(*gen_vp9_cbr_parameters(spec))
-  def test(self, case, gop, bitrate, fps, refmode, looplvl, loopshp):
-    self.init(spec, case, gop, bitrate, fps, refmode, looplvl, loopshp)
+  @parametrize_with_unused(*gen_vp9_cbr_parameters(spec), ['slices'])
+  def test(self, case, gop, bitrate, fps, slices, refmode, looplvl, loopshp):
+    self.init(spec, case, gop, bitrate, fps, slices, refmode, looplvl, loopshp)
```

Comment:
If slices is unused, then it does not need to be passed to "init"

Reply:
updated